File: e2eMain.py
# -*- coding: utf-8 -*-
"""
Created on Thu Aug 30 14:31:31 2018

@author: Balaji_KrishnaIyer
"""

%reset -f

import datetime
PgmStart = datetime.datetime.now()
print("Execution started...",PgmStart,"\n")

DatasetFolder = "D:/DataScienceCapstonePy"
#DatasetName = "DSCapstoneR"

myScriptsDir = DatasetFolder + "/Scripts"
myRawDataDir = DatasetFolder + "/RawData"
myRefDataDir = DatasetFolder + "/RefData"
mySampleDataDir = DatasetFolder + "/SampleDataTest"
myGenDataDir = DatasetFolder + "/GenData"
myReportsDir = DatasetFolder + "/Reports"

import os
os.chdir(myScriptsDir)

#import Test
#exec(open("Test.py").read())

iteration=1
import e2e
exec(open("e2e.py").read())

PgmEnd = datetime.datetime.now()
print("Execution ended...",PgmEnd,"\n")
ExecTime = PgmEnd - PgmStart
print("Execution time: ", ExecTime,"\n")

File: e2eMain.R
# Main Program that calls the other programs in an order

#DatasetFolder <- "/home/bkrishna2006_gmail_com/"
#DatasetName <- "DSCapstoneR"

#myScriptsDir <- paste0(DatasetFolder,DatasetName,"/Scripts")
#myRawDataDir <- paste0(DatasetFolder,DatasetName,"/RawData")
#myRefDataDir <- paste0(DatasetFolder,DatasetName,"/RefData")
#mySampleDataDir <- paste0(DatasetFolder,DatasetName,"/SampleData")
#myGenDataDir <- paste0(DatasetFolder,DatasetName,"/GenData")
#myReportsDir <- paste0(DatasetFolder,DatasetName,"/Reports")

DatasetFolder <- "D:/DSCapstoneRNew"
DatasetName <- "DSCapstoneR"

myScriptsDir <- paste0(DatasetFolder,"/Scripts")
myRawDataDir <- paste0(DatasetFolder,"/RawData")
myRefDataDir <- paste0(DatasetFolder,"/RefData")
mySampleDataDir <- paste0(DatasetFolder,"/SampleData")
myGenDataDir <- paste0(DatasetFolder,"/GenData")
myReportsDir <- paste0(DatasetFolder,"/Reports")

setwd(myScriptsDir)
iteration = "00"
source("e2e.R")